tox.ini

```diff
@@ -5,6 +5,7 @@ envlist = py37-lint, py35, py36, py37, py38
 basepython = python3.7
 deps = pylint
 commands = pylint --rcfile=.pylintrc ibmcloudant test
+setenv = PYTHONPATH = pylint/checkers
```

Comment:
It would be good to avoid the overwrite the values of `PYTHONPATH`, an extension would be better.
Also to follow by the documentation suggested indentation would be good: https://tox.readthedocs.io/en/latest/config.html#conf-setenv:~:text=setenv(MULTI%2DLINE%2DLIST)

Reply:
My original change had that, both for `env:PYTHONPATH` and separators, but it didn't work. See https://travis-ci.com/github/IBM/cloudant-python-sdk/jobs/469265814#L307 and https://github.com/IBM/cloudant-python-sdk/pull/44/commits/fb3b55c6dd052dccfaf8e0a88e6c6558628d32fb .

IIUC we don't set any pythonpath here, so I just went with simplifying (pruning) it.

> follow by the documentation suggested indentation would be good

What would be the point of that here? That looks merely stylistic (that doesn't conform our current `tox.ini`. (See also https://tox.readthedocs.io/en/latest/example/basic.html#setting-environment-variables )

Reply:
okay.